doc/api/events.md

```diff
@@ -319,7 +319,7 @@ console.log(util.inspect(server.listeners('connection')));
 
 ### emitter.on(eventName, listener)
 
-* `eventName` {string|Symbol} The name of the event.
+* `eventName` {String|Symbol} The name of the event.
```

Comment:
String literal or String object?

In other news, why do String objects exist again?